pkg/menu/menu.go

```diff
@@ -0,0 +1,83 @@
+package menu
+
+import (
+	"os"
+
+	ui "github.com/gizak/termui/v3"
+	"github.com/gizak/termui/v3/widgets"
+)
+
+const menuWidth = 50
+
+// newParagraph will return a paragraph object with given initial text.
+func newParagraph(initText string, border bool, location int, wid int, ht int) *widgets.Paragraph {
+	p := widgets.NewParagraph()
+	p.Text = initText
+	p.Border = border
+	p.SetRect(0, location, wid, location+ht)
+	p.TextStyle.Fg = ui.ColorWhite
+	ui.Render(p)
+	return p
+}
+
+// GetInput will present an input box to user and return the user's input.
+// GetInout will check validation of input using isValid function.
+func GetInput(introwords string, location int, wid int, ht int, isValid func(string) (bool, string)) (string, error) {
+	// intro paragraph is to tell user what need to be input here
+	newParagraph(introwords, false, location, len(introwords)+4, 3)
+	location += 2
+
+	// input paragraph is where user input
```

Comment:
I think for cases like this, the variable name is enough to explain what it is for, and the comment isn't needed. If your code is self-explanatory, then you need fewer comments, your code is cleaner :)

Reply:
Try to find other places where this might be the case